src/rules/eqeqeq.rs

```diff
@@ -47,9 +69,17 @@ impl<'c> Visit for EqeqeqVisitor<'c> {
       } else {
         "expected '!==' and instead saw '!='."
       };
-      self
-        .context
-        .add_diagnostic(bin_expr.span, "eqeqeq", message)
+      let hint = if bin_expr.op == BinaryOp::EqEq {
+        "Use '==='"
+      } else {
+        "Use '!=="
```

Comment:
```suggestion
        "Use '!=='"
```

I didn't notice it at a first glance since it's a really tiny missing 😃

Reply:
Nice spot!